motorsd/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+project(motorsd C)
+
+file(GLOB_RECURSE SOURCES "${CMAKE_CURRENT_LIST_DIR}/src/*.c")
+file(GLOB_RECURSE HEADERS "${CMAKE_CURRENT_LIST_DIR}/include/*.h")
+
+include_directories(include)
+
+set(SOURCES src/daemon.c src/command_parser.c src/motor_invoker.c include/motor_invoker.h)
```

Comment:
We don't need to use `GLOB_RECURSE SOURCES` if we manually specify source files here